### DISCUSSION TITLE
Fishing junk to modify existing Fishing Junk Pool

### DISCUSSION THREAD

**Discussion by klugemonkey** [TechReborn/TechReborn#3133] [Ideas]
Current implementation adds another Pool (second roll) to the minecraft:gameplay/fishing/junk LootTable.  Suggestion is to modify the ModLoot.java code to use 

tableBuilder.modifyPools(builder -> {
   builder.with(rubber);
   builder.with(treeTap);
   builder.with(scrap);
});

This would actually add in these items into the existing Pool as entries, so as not to get more than one item from a fishing catch.

Additionally, if some configuration flag could be added to optionally add builder.with(rubberSapling); for Skyblock worlds with no natural tree generation, this would help.

**Comment by Ayutac** [comment]
> Additionally, if some configuration flag could be added to optionally add builder.with(rubberSapling); for Skyblock worlds with no natural tree generation, this would help.

How? You would still need ton of resources that skyblock doesn't have?
